doc/association-rules.rst

```diff
@@ -0,0 +1,74 @@
+Association Rules
+=================
+
+.. figure:: icons/association-rules.png
+
+Induction of association rules.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Data**
+
+Data set
+
+**Outputs**:
+
+-  **Matching Data**
+
+Data instances matching the criteria.
+
+Description
+-----------
+
+This widget runs several algorithms for induction of association rules: the
+original Agrawal's algorithm for sparse data sets, a modified version by Zupan
+and Demšar, which is more suitable for the usual machine learning data, and, finally,
+an algorithm which induces classification rules where the right-hand side of the
+rule is always a class attribute.
+
+.. figure:: images/association-rules-stamped.png
+
+1. Information on the data set.
+
+2. In *Find association rules* you can set criteria for rule induction:
+
+   -  **Minimal support**: percentage of the entire data set covered
+      by the (entire) rule.
+   -  **Minimal confidence**: proportion of the number of examples which
+      fit the right side among those that fit the left side (consequent/antecedent).
```

Comment:
proportion of the examples that fit the right side (consequent) among those that fit the left side (antecedent).